### ISSUE TITLE
Export Raster from Tropycal Cyclone Hazard?

### ISSUE THREAD

**Issue by geohazard89** [CLIMADA-project/climada_python#15]
Hi All, 

I am having some issues trying to export plots of TropCyclone Class - 

I have tried to export write_Hdf5 command but it seems the file is empty. 

Is there any option to write a geotiff or hdf5, correctly? 

Thank you, 
Jose

**Comment by gabrielaznar** [comment]
Hi Jose,

that's strange. Could you give us more info about your error using write_hdf5? For raster data, you can use the write_raster method, given that the centroids are set up as raster (with the centroids' meta attribute correctly set), see the tutorial about for more info [handling hazards](https://climada-python.readthedocs.io/en/latest/tutorial/climada_hazard_Hazard.html). 

Best,
Gabriela

**Comment by geohazard89** [comment]
Hi Gabriela,

Apologies for my late reply, I am trying to get used to Climada. I have exported raster in tif without any issue.

I have other question, is it possible to export plot_rp_Intensity based on different return periods? 
I have attached my code, number 1.

I have attached a screenshot of my code exporting .hdf and the dataset number 2.  

Thank you

1. Code  ![Screenshot 2020-04-25 at 11 00 15](https://user-images.githubusercontent.com/63537357/80275809-043d1980-86e4-11ea-8129-4aa83b17337c.png)


2. File [tc_irma.h5.zip](https://github.com/CLIMADA-project/climada_python/files/4532657/tc_irma.h5.zip)

2. Code ![Screenshot 2020-04-25 at 10 51 01](https://user-images.githubusercontent.com/63537357/80275618-cb507500-86e2-11ea-9947-0e4b0f5d473e.png)
